Common/Data/Custom/Estimize/EstimizeConsensus.cs

```diff
@@ -191,6 +183,15 @@ public override string ToString()
                    Invariant($"{EndTime:yyyyMMdd} ") +
                    Invariant($"by {Source}");
         }
+
+        /// <summary>
+        /// Indicates if there is support for mapping
+        /// </summary>
+        /// <returns>True indicates mapping should be used</returns>
+        public override bool RequiresMapping()
+        {
+            return true;
```

Comment:
I do like this -- types can declare that they require mapping themselves -- right where it belongs 👍